libs/filamat/src/UibGenerator.cpp

```diff
@@ -117,8 +117,15 @@ UniformInterfaceBlock const& UibGenerator::getPerViewUib() noexcept  {
             .add("nearOverFarMinusNear",    1, UniformInterfaceBlock::Type::FLOAT, Precision::HIGH)
             .add("temporalNoise",           1, UniformInterfaceBlock::Type::FLOAT, Precision::HIGH)
 
+            // Screen-space reflection parameters
+            .add("ssrReprojection",         1, UniformInterfaceBlock::Type::MAT4, Precision::HIGH)
+            .add("ssrThickness",            1, UniformInterfaceBlock::Type::FLOAT)
+            .add("ssrBias",                 1, UniformInterfaceBlock::Type::FLOAT)
+            .add("ssrDistance",             1, UniformInterfaceBlock::Type::FLOAT)
+            .add("ssrEnabled",              1, UniformInterfaceBlock::Type::UINT)
```

Comment:
Could we skip this uniform and just use ssrDistance > 0 for instance?

Reply:
I think so, yes. Good idea.